include/zephyr/bluetooth/mesh/access.h

```diff
@@ -1129,23 +1129,31 @@ int bt_mesh_model_correspond(const struct bt_mesh_model *corresponding_mod,
  */
 bool bt_mesh_model_is_extended(const struct bt_mesh_model *model);
 
-/** @brief Indicate that the composition data will change on next bootup.
+/** @brief Set Composition Data Page contents.
  *
- *  Tell the config server that the composition data is expected to change on
- *  the next bootup, and the current composition data should be backed up.
+ *  Can only be used for pages >= 128. If this is called with a page number
+ *  not present on the device, it will return an error code.
```

Comment:
So it returns success if page number is present but < 128 or error? Not clear from description :)